.github/workflows/test.yml

```diff
@@ -48,6 +48,22 @@ jobs:
           MYSQL_USER: faf-python-server
           MYSQL_PASSWORD: banana
 
+      faf-rabbitmq:
+        image: rabbitmq:3-management-alpine
```

Comment:
Actually not sure if this needs the management plugin. I think probably not since that's just a web interface.